src/Scheduler.ts

```diff
@@ -6,6 +6,6 @@ export interface Scheduler {
   schedule<T>(work: (state?: T) => Subscription | void, delay?: number, state?: T): Subscription;
   flush(): void;
   active: boolean;
-  actions: Action[];
+  actions: Action<any>[]; // XXX: use `any` to remove type param `T` from `Scheduler`.
```

Comment:
I chose this approach to keep compatibility without a breaking change.

Should we add `T` to `Scheduler` for more strongly typing?